### PR TITLE
proposed fixes to sophia.md

### PR DESCRIPTION
**Pull request by hdiedrich**
Hi Rob,

these are all proposals, sometimes questions. In case it helps.

Best,
Henning

### CODE REVIEW

File: contracts/sophia.md [hunk 1]
```diff
@@ -232,13 +257,17 @@ Record fields access is written `r.f` and map lookup `m[k]`. For instance,
 Looking up a non-existing key in a map results in the contract failing. See
 `Map.member` and `Map.lookup` below for safe lookups.
 
+[the entire contract or only this call into this contract?]
```
Comment (UlfNorell) [inline contracts/sophia.md]: The latter. Should say "the contract *call*".

File: contracts/sophia.md [hunk 2]
```diff
@@ -268,12 +297,12 @@ The following builtin functions are defined on maps:
...
 
-### Builtins
+### Built-ins
 
 #### Account interface
 
-To spend tokens from the contract account to the account "to" you call the `Chain.spend` function.
-
+To spend tokens from the contract's implicit account to the account "to", you
+call the `Chain.spend` function. The amount is in eto Aeons:
```
Comment (UlfNorell) [inline contracts/sophia.md]: eto -> atto (10<sup>-18</sup>)

File: contracts/sophia.md [hunk 3]
```diff
@@ -559,7 +593,7 @@ Decl ::= 'contract' Con '=' Block(Decl)
        | 'type'     Id ['(' TVar* ')'] ['=' TypeAlias]
        | 'record'   Id ['(' TVar* ')'] '=' RecordType
        | 'datatype' Id ['(' TVar* ')'] '=' DataType
-       | Modifier* 'function' Id ':' Type
+       | Modifier* 'function' Id [':' Type]
```
Comment (UlfNorell) [inline contracts/sophia.md]: No, abstract function declarations without a definition really do need a type signature.

**Comment by evdoks** [comment]
@hdiedrich Are you still planning to keep working on this PR?

**Comment by ThomasArts** [comment]
Closing this commit, Sophia has changed since.

**Pull request closed without merging**